platform/android/vulkan_context_android.h

```diff
@@ -1,5 +1,5 @@
 /*************************************************************************/
-/*  vk_renderer_jni.cpp                                                  */
+/*  vulkan_context_android.h                                             */
```

Comment:
It's likely that more vulkan logic will be added inside and outside of this class. Can you restore the `vulkan` directory and keep all android platform vulkan specific logic (including this class) in there.